pkg/certifier/osv/osv.go

```diff
@@ -157,35 +157,46 @@ func generateDocument(purl string, digest []string, vulns []osv_scanner.Entry) (
 	return doc, nil
 }
 
-func createAttestation(purl string, digest []string, vulns []osv_scanner.Entry) *attestation_vuln.VulnerabilityStatement {
+func createAttestation(projectURL string, digests []string, vulns []osv_scanner.Entry) *attestation_vuln.VulnerabilityStatement {
```

Comment:
any reason why this was changed to `projectURL`? A purl is a "package URL"

Reply:
purl is fine here :)

Reply:
Sorry, I assumed that `purl` was `projectURL`. I am going to change it to `packageURL` because `purl` is not very descriptive, and kind of vague.